run.sh

```diff
@@ -143,12 +147,17 @@ fi
 
 # Download genesis
 if [ "$DOWNLOAD_GENESIS" == "1" ]; then
-  echo "Downloading genesis"
+  echo "Downloading genesis $GENESIS_URL"
   curl -sfL $GENESIS_URL > genesis.json
+  # Ensure we are using the block that contains 'genesis_time' on the first level
+  # Some chains might be using an encapsulated response (e.g. a jsonrpc embodied response)
+  cat genesis.json | jq '.. | objects | . as $parent | with_entries(select(.key  == "genesis_time")) | select(. != {})| $parent ' > genesis_parsed.json
```

Comment:
Is this safe to do for all chains?

Reply:
I would say it is safe. What this command does is to ensure that the "genesis_time" key is on the base level of the json. And if its not found on the base level it will convert the json to ensure it will be. Basically it ensures it follows the recommended JSON schema of the genesis.json